applications/nrf_desktop/configuration/nrf52840gmouse_nrf52840/sysbuild_mcuboot_smp.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+
+SB_CONFIG_BOOTLOADER_MCUBOOT=y
+SB_CONFIG_BOOT_SIGNATURE_TYPE_RSA=y
+SB_CONFIG_BOOT_SIGNATURE_KEY_FILE="\${APPLICATION_CONFIG_DIR}/images/mcuboot/mcuboot_private.pem"
```

Comment:
It used to be `CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256` here

Reply:
Oh, it is an only config that has it set, must have missed that. Thanks